Enterprise-CMCS/eAPD
# 4558 [Feature] Update Read-Only View for MMIS changes to Executive Summary

Resolves #4558 

### Description
Updates to the Export view's Executive Summary section for MMIS APDs.

### Chromatic Link
https://www.chromatic.com/build?appId=61d5b948cf6f17003a12bf77&number=1456

### Significant changes or possible side effects
- Removed the description text below the title in the Executive Summary for any APD type.
- For MMIS types: Summary of Activity will use the text from activity snapshot
- Executive Summary Budget section will display No Table message if both DDI and M&O tables are hidden

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| MMIS APD Export View Program Budget Tables section | no budget data is present for DDI | DDI table is NOT visible | jest |
| MMIS APD Export View Program Budget Tables section | no budget data is present for M&O | M&O table is NOT visible | jest |
| MMIS APD Export View Program Budget Tables section | no tables are visible | No tables message displays | jest |
| selectBudgetExecutiveSummary selector | activity summary data available | data selected contains activity summary as summary | jest |
| selectBudgetExecutiveSummary selector | no activity summary data available; activity snapshot data available | data selected contains activity snapshot as summary | jest |

### Steps to manually verify this change

1. For MMIS APD without any budget data, go to Export & Submit > Continue to Review and verify that the Program Budget Tables section displays the No Table message (neither the DDI table nor M&O table are present)
2. For MMIS APD with budget data, go to Export & Submit > Continue to Review and verify that the Program Budget Tables section displays the DDI and/or M&O tables depending on what budget data exists for the chosen APD.
3. For any MMIS APD, go to Export & Submit > Continue to Review and verify that below each Activity name, the same text inside the APD Overview snapshot displays.

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when QA has


### This pull request can be merged when
